panel/src/components/Views/Pages/index.js

```diff
@@ -1,9 +1,11 @@
 import PageView from "./PageView.vue";
+import PageChangesView from "./PageChangesView.vue";
 import SiteView from "./SiteView.vue";
 
 export default {
 	install(app) {
 		app.component("k-page-view", PageView);
+		app.component("k-page-changes-view", PageChangesView);
```

Comment:
`Todo:` We also need it for the site view.

Reply:
I was wondering what we do with that and also for files. Would it be confusing if we show the home page in site view and the parent page for files? Or is it more confusing if we don't have the comparison view for those at all.

Reply:
I'd say for site, yes the home page - as where the normal preview button also sends the user. I think for files and users it's ok to not have one as it's difficult to pick one that really makes sense in most setups.